Week1/homework/Create-table.js

```diff
@@ -0,0 +1,71 @@
+var mysql = require('mysql');
+var connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'userdb',
+});
+
+connection.connect();
+
+var world_db = 'CREATE DATABASE IF NOT EXISTS World ';
+connection.query(world_db, function(error, result) {
+  if (error) throw error;
+  console.log('Created Database ');
+});
+
+connection.query('USE World');
+
+var create_tables = [
+  'CREATE TABLE IF NOT EXISTS countries (country_id INT , country_name VARCHAR(50), country_population INT, continent VARCHAR(50),surface_areas INt) ',
+  'CREATE TABLE IF NOT EXISTS cities (city_id INT , city_name VARCHAR(50),city_population INT) ',
+];
+
+for (var i in create_tables) {
+  connection.query(create_tables[i], function(error, results) {
+    if (error) throw error;
+    console.log('Created new table');
+  });
+}
+
+var countries = [
```

Comment:
You can do this, or take the data part out and make use of this helper https://github.com/mysqljs/mysql#escaping-query-identifiers